docs/data/joy/getting-started/templates/sign-in-side/App.tsx

```diff
@@ -180,11 +181,11 @@ export default function JoySignInSideTemplate() {
             >
               <FormControl required>
                 <FormLabel>Email</FormLabel>
-                <Input placeholder="Enter your email" type="email" name="email" />
+                <Input type="email" name="email" />
               </FormControl>
               <FormControl required>
                 <FormLabel>Password</FormLabel>
-                <Input placeholder="•••••••" type="password" name="password" />
+                <Input type="password" name="password" />
```

Comment:
These placeholders feel confusing. I can't find a serious benchmark that has them. (Stripe, Figma)